src/service/resources-impl.js

```diff
@@ -1628,16 +1655,15 @@ export class Resource {
   /**
    * Requests the resource's element to be built. See {@link AmpElement.build}
    * for details.
-   * @param {boolean} force
    * @return {boolean}
```

Comment:
Well, in this case you are correct. We can preserve return value here since it's actually meaningful in this case. However, it's definitely no longer meaningful for `ElementProto.build` and should be removed there.